src/assets/AssetsContractController.test.ts

```diff
@@ -64,6 +66,13 @@ describe('AssetsContractController', () => {
     );
   });
 
+  it('should throw not a smart contract error when attempting to get token standard and details for an EOA', async () => {
```

Comment:
Nit: I didn't understand this properly until reading it for the third time :sweat_smile:  I thought it was saying it should NOT throw an error.

```suggestion
  it('should throw "not a smart contract" error when attempting to get token standard and details for an EOA', async () => {
```